data.sql

```diff
@@ -10,4 +10,27 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 VALUES (3, 'Pikachu', '2021-01-07', 1, FALSE, 15.04);
 
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES (4, 'Devimon', '2017-05-12', 5, TRUE, 11.00);
\ No newline at end of file
+VALUES (4, 'Devimon', '2017-05-12', 5, TRUE, 11.00);
+
+-- ALTER ANIMALS TABLE
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (5, 'Charmander', '2020-02-08', 0, FALSE, -11);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (6, 'Plantmon', '2021-11-15', 2, TRUE, -5.7);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (7, 'Squirtle', '1993-04-02', 3, FALSE, -12.13);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (8, 'Angemon', '2005-05-12', 1, TRUE, -45);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (9, 'Boarmon', '2005-05-07', 7, TRUE, 20.4);
```

Comment:
* According to the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Animal%3A%20His%20name%20is%20Angemon,tried%20to%20escape%207%20times.) the date of birth is june(06) and not may(05) 
![image](https://github.com/tajulafreen/Vet-Clinic/assets/52497246/07a976d9-e866-4a20-8188-88323a2e5b7e)